metron-interface/metron-alerts/src/app/utils/constants.ts

```diff
@@ -38,5 +38,6 @@ export let TREE_SUB_GROUP_SIZE = 5;
 export let DEFAULT_FACETS = ['source:type', 'ip_src_addr', 'ip_dst_addr', 'host', 'enrichments:geo:ip_dst_addr:country'];
 export let DEFAULT_GROUPS = ['source:type', 'ip_src_addr', 'ip_dst_addr', 'host', 'enrichments:geo:ip_dst_addr:country'];
 export let INDEXES =  environment.indices ? environment.indices.split(',') : [];
+export let POLLING_DEFAULT_STATE = environment.defaultPollingState;
```

Comment:
What does the POLLING_DEFAULT_STATE setting do?

Reply:
Added info for this in Readme and CC

Reply:
Would it be possible to change this to a better name?  Something like DISABLE_POLLING?  Setting POLLING_DEFAULT_STATE to true to stop polling doesn't make sense to me.